$MirOS: ports/mailnews/pine/patches/patch-imap_src_ipopd_Makefile,v 1.1.7.1 2005/03/18 15:49:48 tg Exp $
--- imap/src/ipopd/Makefile.orig	Tue Oct 24 23:54:45 2000
+++ imap/src/ipopd/Makefile	Wed May  4 23:28:07 2005
@@ -19,7 +19,7 @@
 
 
 C = ../c-client
-CCLIENTLIB = $C/c-client.a
+CCLIENTLIB = -L$C -lc-client
 SHELL = /bin/sh
 
 # Get local definitions from c-client directory
